content/blog/2021/11/camel-k-release-1-7/index.md

```diff
@@ -0,0 +1,106 @@
+
+In the last release we've dedicated some time to include improvements about the way you can customize your Camel K installation on a Kubernetes cluster.
+
+### Kustomize
+
+Let's start with the introduction of [Kustomize](https://kustomize.io/). Here the extract of the feature to be soon documented officially:
+
+The `config` directory contains all the resources for installation and configuration of the camel-k operator. While, by default, those resource are installed through the `kamel` binary, they can also be applied directly to a cluster using `kustomize`. These resources can be modified prior to their installation and the `kustomize.yaml` files be changed to **include extra patches and settings**, as required.
+
+More details on how to use it can be found in the [release page](https://github.com/apache/camel-k/releases/tag/v1.7.0).
+
+### Custom labels
+
+Sometimes you may be interested to **introduce certain custom labels** to all the `Pods` running for all your `Integrations`. In such circumstances, you can build the Camel K operator on your own and leverage the presence of a new `Golflag` to set certain information that will be configured and then executed from the operator:
+```
+GOLDFLAGS += -X github.com/apache/camel-k/pkg/util/label.AdditionalLabels=department=Accounting;version=1.2.3;int-name=token_integration_name
+```
+With this simple configuration, you can let each `Integration` to inherit those `Labels` automatically, withouth having to declare them every time. Discover more information about the [custom label feature](https://github.com/apache/camel-k/pull/2646).
+
+## Polished Integration status
+
+This one is a series of improvements to **include more descriptive information** related to the status of the integration. In particular we've worked to a couple of features that introduced a more detailed error reporting in the `Integration` and more reliability about the scaling status.
+
+## Secondary Integration Platform
+
+So far, a single operator was able to manage the `Integration` created on a namespace. This simple model was a bit limiting, as we were not able to distinguish a way to have more than one operator on the same namespace. With the new feature developed, now you will be able to declare a secondary `Integration Platform` and **have more than one operator** collaborating on the same `Namespace`. This is an advanced feature that will let you unlock more advanced deployment topology in order to have a [more scalable and resilient configuration](/camel-k/next/architecture/advanced.html).
+
+## Traits enhancements
+
+We have worked to a few interesting enhancements on the following traits:
+
+* [Health](/camel-k/next/traits/health.html)
+* [Camel](/camel-k/next/traits/camel.html)
+* [Route](/camel-k/next/traits/route.html)
+
+The new `health` trait was introduced to replace the responsibility of exposing the endpoint from `container` trait, encapsulating its responsibility in this self contained new trait.
```

Comment:
Maybe we can mention that now it's possible to enable liveness and readiness probes individually.

Reply:
Mmm, I cannot find any reference of that in the release page. Do you mind pointing me at the GH issue/PR?

Reply:
Here is the issue https://github.com/apache/camel-k/issues/1610, which I fixed as part of https://github.com/apache/camel-k/pull/2740 but forgot to mention in the release notes :)